scripts/sign

```diff
@@ -7,6 +7,15 @@ sign_and_submit() {
     (source AMOKEYS && (web-ext sign -s build --api-key $AMOKEY --api-secret $AMOSECRET || true))
```

Comment:
`web-ext sign -s build --api-key $AMOKEY --api-secret $AMOSECRET` needs to have `-a web-ext-artifacts/nonewtab` added to it for the newtab build otherwise the newtab vs nonewtab builds will overwrite each other :)

Reply:
Fixed.
Wouldn't the same problem appear with beta and stable? Or does stable have a different output name?

Reply:
I guess it does? I've never noticed any problems. I'm not sure it outputs an xpi locally because the stable build is distributed on the AMO and is automatically uploaded in that step. 🤷‍♂️